colcon-build/action.yaml

```
@@ -19,6 +19,10 @@ inputs:
     description: ""
     required: false
     default: ${{ github.token }}
+  rosdep-update-arguments:
+    description: ""
+    required: false
+    default: ""
```

If there is no other candidate `rosdep` args, renaming the input would be better.

```suggestion
  include-eol-distros:
    description: ""
    required: false
    default: false
```